src/solvers/imex_ark.jl

```diff
@@ -53,60 +55,84 @@ function step_u!(integrator, cache::IMEXARKCache)
     s = length(b_exp)
 
     if !isnothing(T_imp!) && !isnothing(newtons_method)
-        update!(
-            newtons_method,
-            newtons_method_cache,
-            NewTimeStep(t),
-            jacobian -> isnothing(γ) ? sdirk_error(name) : T_imp!.Wfact(jacobian, u, p, dt * γ, t),
-        )
+        @nvtx "update!" color = colorant"yellow" begin
```

Comment:
Can we just use `NVTX.@range` directly?

Reply:
Yes, I saw this helper in clima Atmos, I just thought it was useful to reduce some boiler plate

Reply:
Also, just leave the `color` option off by default (it generates random colors, which will be easy to distinguish)